src/cephadm/cephadm.py

```diff
@@ -5502,6 +5509,20 @@ def _get_parser():
         help='Configuration input source file',
     )
 
+    parser_set_coredump_max_size = subparsers_orch.add_parser(
+        'set-coredump-max-size',
```

Comment:
For future-proofing maybe consider naming this `set-coredump-config` and rename `--size` below as `--max-size`. Then if someone asks for more tunables regarding core dumps (let's use compression as an example we could just add `--compress=yes|no` down the road w/o needing a new command.